org.mixedrealitytoolkit.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -28,12 +28,43 @@ public class GazePinchInteractor :
         [Tooltip("The hand controller used to get the selection progress values")]
         private ArticulatedHandController handController;
 
+        #region Properties and methods from deprecated XRBaseController
+
+        XRControllerState m_ControllerState;
+        /// <summary>
+        /// The current state of the controller.
+        /// </summary>
+        public XRControllerState currentControllerState
```

Comment:
I'm pretty sure all controller types, including XRControllerState, were deprecated by Unity.  We shouldn't be adding "controllers state" to an interactor if deprecated.

Reply:
Also I don't understand the need to even add this state object as public.   Let's not add this, as I see no usage.

Reply:
> I'm pretty sure all controller types, including XRControllerState, were deprecated by Unity. We shouldn't be adding "controllers state" to an interactor if deprecated.

I thought the same first but then I noticed that XRControllerState is not mentioned as part of the deprecation in Unity's changelog https://docs.unity3d.com/Packages/com.unity.xr.interaction.toolkit@3.0/changelog/CHANGELOG.html ... Also Visual Studio doesn't suggests it is deprecated as in:
<img width="488" alt="image" src="https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/ca2b8279-a46e-4c2d-8a36-6fe65be5161a">

> Also I don't understand the need to even add this state object as public. Let's not add this, as I see no usage.

I kept it public because that's how it is exposed in the deprecated XRController, my reasoning to keep it this way is that if we are going to remove XRController as part of XRI3 migration but some of our customers have code already depending on that property then helping them to upgrade their code to consume the replacement property(ies) would be easier if they have the same signature as they currently have in XRController.

@AMollis Given the above ^ explanation, Do you still want me to remove it or make it private?

Reply:
As per our offline discussion, I have avoided using the XRControllerState in GazePinchInteractor.  This is done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/743/commits/07d41155987a3c194dbeb0c7404455a797418ada , resolving.